corehq/apps/users/templates/users/roles_and_permissions.html

```diff
@@ -118,6 +118,9 @@ <h3 class="panel-title">
           </th>
           {% endif %}
           <th>{% trans "Actions" %}</th>
+          {% if is_managed_by_upstream_domain %}
+          <th>{% trans "Local App Config" %}</th>
```

Comment:
Maybe "Local Project Config" or "Locally configured"?  This identifies which roles are local and can thus be managed, right?